### DISCUSSION TITLE
Swaguer UI

### DISCUSSION THREAD

**Discussion by DeltaCorporate** [directus/directus#11992] [Feature Requests]
It will be very cool to have the possibility to choose it's own primary and secondary colors. And it will be also very cool to have a plugin market place where we can find many plugins and especially a swagguer ui plugin or anu other visual platform to easily create it's own documentation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
